src/components/apartments/apartment/Apartment.css

```diff
@@ -8,13 +8,33 @@
 
 .photoContainer {
   height: 58px;
+  box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px, rgba(60, 64, 67, 0.15) 0px 2px 6px 2px;
 }
 
 .apartment-photo {
+ 
   width: 116px;
   height: 100%;
   background-size: cover;
   background-repeat: no-repeat;
+  z-index: -1;
+}
+
+.apartment-photo .apartment-unit {
+  position: relative;
+  top: 0px;
+  left: 6px;
+  font-size: 8px;
```

Comment:
This is too small. Material design says we shouldn't get smaller than 14px but I say we go 10px for this. 

https://learnui.design/blog/mobile-desktop-website-font-size-guidelines.html#:~:text=Now%2C%20each%20font%20is%20different,font%20size%20to%20start%20with.

Reply:
so are we not following the font size from our design?